scio-elasticsearch/src/main/java/org/apache/beam/sdk/io/elasticsearch/ThrowingConsumer.java

```diff
@@ -18,8 +18,9 @@
 package org.apache.beam.sdk.io.elasticsearch;
 
 import java.io.Serializable;
-import java.util.function.Consumer;
 
-public interface SerializableConsumer<T> extends Consumer<T>, Serializable {
+@FunctionalInterface
+public interface ThrowingConsumer<T> extends Serializable {
 
```

Comment:
@regadas Maybe should we make this more specific by changing above to:
```
public interface ThrowableConsumer<T extends BulkExecutionException> extends Serializable {}
```
For given function, it would be useful to be able to access methods specific to BulkExecutionException? As this function is *only* applied with this specific exception type. In that was you can just throw T as well.